src/braket/circuits/composite_operators.py

```diff
@@ -72,9 +72,17 @@ class QFT(CompositeOperator):
 
     Args:
         qubit_count (int): Number of target qubits.
+        method (bool): Optional boolean flag for decomposition,
```

Comment:
Method below is a string, not a boolean. Also update the docstring to reflect what valid values are.